main.js

```diff
@@ -42,9 +42,11 @@
       })
       .sort(function(a, b) {
         // Start the list at midnight regardless of the user's timezone
-        if (a.timeString < b.timeString) return -1;
-        if (a.timeString > b.timeString) return 1;
-        return 0;
+        var tl = [a, b].map(function(x) {
+          var d = x.dateTime;
+          return (d.getHours() * 60 + d.getMinutes());
+        });
+        return (tl[0] - tl[1]);
```

Comment:
This getHours + getMinutes calculation is a great shout, thanks!

fyi I'd have probably done this calculation in `calculateEventTimes` so that you don't need to run a `map` inside the sort loop - you could just do something like this:
```javascript
.sort(function(a, b) {
  // Start the list at midnight regardless of the user's timezone
 return a.timeNumber - b.timeNumber;
})
```
which would be a little simpler. But this works so let's ship it.

Reply:
Yeah that's a great point! I was trying to limit the change to just the sorting part without modifying the broader data structure, but doing it in calculateEventTime would certainly be more efficient. I can do another pull request! :-)